### PR TITLE
⚒️　Fixing the toolbar display bug

### PR DESCRIPTION
**Pull request by FunamaYukina**
## Summary

ss 2592

### CODE REVIEW

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.module.css [hunk 1]
```diff
@@ -45,3 +45,7 @@
 .toolbarWrapper > * {
   pointer-events: auto;
 }
+
+:global([data-state='expanded']) + .main .toolbarWrapper {
+  display: none;
+}
```
Comment (kumanoayumi) [inline frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.module.css:51]:
```suggestion
@media screen and (max-width: 768px) {
  :global([data-state='expanded']) + .main .toolbarWrapper {
    opacity: 0;
    visibility: hidden;
  }
}
```

Comment (kumanoayumi) [reply frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.module.css:51]: For devices with widths other than the Mobile version, I would like to see the toolbar displayed, so I would also like the following added to media query of the toolbarWrapper.

```
@media screen and (max-width: 768px) {
  .toolbarWrapper {
    transition: opacity 0.3s, visibility 0.3s;
    opacity: 1;
    visibility: visible;
    min-width: 354px;
  }
}
```

Setting min-width reduces layout wobble when scaling down.

**Review by kumanoayumi — CHANGES_REQUESTED** [review]
Please check the suggest comments! 😸

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.module.css (hunk 1 above)
Comment (FunamaYukina) [reply frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.module.css]: Thank you for your review and code🙏. I've implemented your suggestions, and the toolbar is now functioning much more smoothly✨. Thank you very much for your assistance.

fixed: https://github.com/liam-hq/liam/pull/359/commits/1c20fd1abe6b205c53a123c1b7487afd68e6532e

**Review by MH4GF — APPROVED** [review]
LGTM 👍🏻

**Review by kumanoayumi — APPROVED** [review]
thx! 🌟 LGTM 👍

**Pull request merged**